requirements.txt

```diff
@@ -5,3 +5,4 @@ mock==1.3.0
 nose==1.3.4
 python_swiftclient==2.7.0
 rednose==1.1.1
+six
```

Comment:
Please add a concrete version to `requirements.txt` Also, `six` will need to be added to the `setup.py` as well.